helita/sim/bifrost.py

```diff
@@ -268,10 +317,95 @@ def _init_vars(self, *args, **kwargs):
                               self.zdn.astype(rdt), self.dzidzup.astype(rdt),
                               self.dzidzdn.astype(rdt))
 
-    def get_var(self, var, snap=None, *args, **kwargs):
+    def get_varTime(self, var, snap=None, iix=None, iiy=None, iiz=None,
+                    order='F', mode='r', *args, **kwargs):
```

Comment:
Needs better docstring (parameters, usage, etc.). `iix`, `iiy`, `iiz` are terrible variable names. They should be changed to something more meaningful.